microsoft-edge/devtools-guide-chromium/speed/get-started.md

```diff
@@ -22,440 +22,480 @@ ms.date: 05/04/2021
+1. In Visual Studio Code, open the `/travel-site/index.html` file.
 
-1. Select the **Network** tool.
+1. Near the top of the file, find the list of `<link>` tags that load the stylesheets on the page.
 
-1. Press `Ctrl`+`Shift`+`P` (Windows, Linux) or `Command`+`Shift`+`P` (macOS) to open the Command Menu again.
+1. Delete the two lines of code that load the `contact-form.css` and `gallery.css` files:
 
-1. Start typing `blocking`, and then select **Show Request Blocking**.
+    ![VS Code, with the index.html opened, showing where the two link tags are](./images/delete-unused-css.png)
 
-   ![The Request blocking tool](../media/speed-glitch-tony-remix-updated-network-drawer-request-blocking-empty.msft.png)
+1. Save your changes in Visual Studio Code, then refresh the webpage in Microsoft Edge, and run a new audit in the **Lighthouse** tool again to see how your changes affect the load performance.
 
-1. Click **Add Pattern** (![Add Pattern](../media/add-pattern-icon.msft.png)), type `/libs/*`, and then press `Enter` to confirm.
+###### Automatically remove non-critical CSS
 
-   ![Add a pattern to block any request to the libs directory](../media/speed-glitch-tony-remix-updated-network-drawer-request-blocking-added.msft.png)
+In the previous step, your score improved slightly but **Lighthouse** still flags other CSS files as blocking the initial render of the page.
 
-1. Refresh the page.  The jQuery and Lodash requests are red, meaning that the requests were blocked.   The page still loads and is interactive, so it looks like these resources aren't needed whatsoever!
+The webpage uses the remaining CSS files so you can't remove them. However, it's possible to split them in two groups:
 
-   ![The Network panel shows that the requests have been blocked](../media/speed-glitch-tony-remix-updated-network-reloaded-drawer-request-blocking-added.msft.png)
+* Critical CSS code that needs to block the rendering of the webpage because it visually impacts the style and layout of the part of the webpage that users see when it loads.
 
-1. Click **Remove all patterns** (![Remove all patterns](../media/remove-icon.msft.png)) to delete the `/libs/*` blocking pattern.
+    For example, the title on the webpage uses the `header h1` CSS rule in the `/travel-site/assets/base.css` file.
 
-In general, the **Request blocking** tool is useful for simulating how your page behaves when any given resource isn't available.
+* Non-critical CSS code used to render parts of the page that aren't visible when the page loads.
 
-Now, remove the references to these files from the code and audit the page again:
+    For example, the `/travel-site/assets/desktop.css` file is only needed when the viewport is larger than `665px`.
 
-1. Back in the editor tab, open `template.html`.
+To automatically split your CSS code this way, you can use the _Critical_ tool. To learn more, see the [Critical project repository](https://github.com/addyosmani/critical) at GitHub.
```

Comment:
```suggestion
To automatically split your CSS code this way, you can use the _Critical_ tool. To learn more, see the [Critical project repo](https://github.com/addyosmani/critical).
```
globally i'm omitting "at github" when say "see repo", b/c "repo" already probably means GitHub.

i think "repo" is clearer than "repository", which is almost an unfamiliar word, like "enum" vs "enumeration".  corp style guide's principle of "be informal" can support, IMO.